src/components/CommentSummaryButton/index.tsx

```diff
@@ -12,9 +12,9 @@ import CommentSummary from '../CommentSummary';
 import PopoverButton from '../PopoverButton';
 import styles from './styles.module.scss';
 
-export type PublicProps = {};
+export type PublicProps = Record<string, unknown>;
 
-export type DefaultProps = {};
+export type DefaultProps = Record<string, unknown>;
```

Comment:
Why do we even need `PublicProps` and `DefaultProps`? It seems odd that we have them and set them to `{}	`.

Reply:
That's what we've done in the past. It means "there is no default props" or "no public props". I don't think we necessarily need the `DefaultProps` all the time but having `PublicProps` and `InternalProps` makes sense to me–one can quickly scan a component file and see the different props.

Reply:
This is valid in TS, it's just the linter that does not like `{}`

Reply:
I removed this `DefaultProps` type.